src/services/emailService.js

```
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: Number(process.env.SMTP_PORT || 587),
```

Transporter configuration uses `process.env.SMTP_HOST` and `process.env.SMTP_PORT` directly. For local dev/tests, defaulting `SMTP_PORT` to 587 may be surprising; earlier project code used port 1025 as a safe local default. Provide sensible defaults or a dry-run fallback to avoid silent failures. Also consider skipping creating a transporter when NO_EMAIL=1.